scripts/manual_review_analyses.py

```diff
@@ -21,18 +21,58 @@ def get_analysis_data(client, type, id):
         return client.get_inheritance_analyses(id)
 
 
-def post_variants_export(client, type, id):
-    if type == 'PATIENT':
-        return client.post_patient_analyses_variants_export(id)['exportId']
-    elif type == 'INHERITANCE':
-        return client.post_inheritance_analyses_variants_export(id)['exportId']
-
-
-def get_variants_export(client, type, id, export_id):
-    if type == 'PATIENT':
-        return client.get_patient_analyses_variants_export(id, export_id)
-    elif type == 'INHERITANCE':
-        return client.get_inheritance_analyses_variants_export(id, export_id)
+def post_variants_export(client, analysis_type, variant_type, analysis_id):
+    if analysis_type == 'PATIENT':
+        if variant_type == 'molecular_variant':
+            return client.post_patient_analyses_variants_export(analysis_id)['exportId']
+        elif variant_type == 'copy_number_variation':
+            return client.post_patient_analyses_cnv_export(analysis_id)['exportId']
+    elif analysis_type == 'INHERITANCE':
+        if variant_type == 'molecular_variant':
+            return client.post_inheritance_analyses_variants_export(analysis_id)['exportId']
+        elif variant_type == 'copy_number_variation':
+            return client.post_inheritance_analyses_cnv_export(analysis_id)['exportId']
+
+
+def get_variants_export(client, analysis_type, variant_type, analysis_id, export_id):
+    if analysis_type == 'PATIENT':
+        if variant_type == 'molecular_variant':
+            return client.get_patient_analyses_variants_export(analysis_id, export_id)
+        elif variant_type == 'copy_number_variation':
+            return client.get_patient_analyses_cnv_export(analysis_id, export_id)
+    elif analysis_type == 'INHERITANCE':
+        if variant_type == 'molecular_variant':
+            return client.get_inheritance_analyses_variants_export(analysis_id, export_id)
+        elif variant_type == 'copy_number_variation':
+            return client.get_inheritance_analyses_cnv_export(analysis_id, export_id)
+
+
+def get_variants(client, analysis_type, variant_type, analysis_id, variant_count):
+    export_id = post_variants_export(client, analysis_type, variant_type, analysis_id)
+    export = None
+    while export is None:
+        try:
+            time.sleep(variant_count/100 + 1)  # delay to request exported report, min 1 sec.
+            export = get_variants_export(client, analysis_type, variant_type, analysis_id, export_id)
+        except HTTPError:
+            pass
+
+    return export
+
+
+def count_manual_review_labels(variants_export):
+    manual_review_count = [0, 0, 0]  # manual_review labes Y, Y2, Y3 (rare)
```

Comment:
What is the purpose of this list?
Could a dictionary be an alternative?

Reply:
purpose, store the counts while looping through the variants. Yes dictionary could be an alternative, but would make the code a bit more complex.